builtins/amp-ad.js

```diff
@@ -292,7 +292,9 @@ export function installAd(win) {
      */
     getAdCid_() {
       const scope = clientIdScope[this.element.getAttribute('type')];
-      if (!scope) {
+      const consentId = this.element.getAttribute(
+          'data-consent-notification-id');
+      if (!(scope || consentId)) {
         return Promise.resolve();
       }
       return cidForOrNull(this.getWin()).then(cidService => {
```

Comment:
When `!scope`, why do we need to query `cidFor`? Does it add any real cost on the other hand?